bindings/python/CONTRIBUTING.md

```diff
@@ -0,0 +1,388 @@
+# Contributing to PyMongoArrow
+
+Contributions to PyMongoArrow are always encouraged. Contributions can be as simple
+as minor tweaks to the documentation. Please read these guidelines
+before sending a pull request.
+
+## Bugfixes and New Features
+
+Before starting to write code, look for existing
+[tickets](https://jira.mongodb.org/browse/ARROW) or [create
+one](https://jira.mongodb.org/browse/ARROW) for your specific issue or
+feature request. That way you avoid working on something that might not
+be of interest or that has already been addressed.
+
+## Supported Interpreters
+
+PyMongoArrow supports CPython 3.7+ and PyPy3.8+. Language features not
+supported by all interpreters can not be used.
+
+## Style Guide
+
+PyMongo follows [PEP8](http://www.python.org/dev/peps/pep-0008/)
+including 4 space indents and 79 character line limits.
+
+## General Guidelines
+
+-   Avoid backward breaking changes if at all possible.
+-   Write inline documentation for new classes and methods.
+-   Write tests and make sure they pass (make sure you have a mongod
+    running on the default port, then execute `tox -e test` from the cmd
+    line to run the test suite).
+-   Add yourself to doc/contributors.rst `:)`
+
+## Authoring a Pull Request
+
+**Our Pull Request Policy is based on this** [Code Review Developer
+Guide](https://google.github.io/eng-practices/review)
+
+The expectation for any code author is to provide all the context needed
+in the space of a pull request for any engineer to feel equipped to
+review the code. Depending on the type of change, do your best to
+highlight important new functions or objects you've introduced in the
+code; think complex functions or new abstractions. Whilst it may seem
+like more work for you to adjust your pull request, the reality is your
+likelihood for getting review sooner shoots up.
+
+**Self Review Guidelines to follow**
+
+-   If the PR is too large, split it if possible.
+
+    - Use 250 LoC (excluding test data and config changes) as a
```

Comment:
-    - Use 250 LoC (excluding test data and config changes) as a
+    - Use 250 lines of code (excluding test data and config changes) as a
             rule-of-thumb.

(Awful acronym)